### ISSUE TITLE
Issue with SoftDevice 132

### ISSUE THREAD

**Issue by ekawahyu** [micooke/SoftwareSerial#2]
I am having issue with SoftDevice 132, Sandeep Mistry fork. If I build the sketch without SoftDevice, the SoftwareSerial seems to work just fine. But I see some gibberish output for the same sketch with SoftDevice. Is this a known issue, or am I missing anything? Please let me know if you need further info, thanks.

**Comment by micooke** [comment]
Can you link it? I'm not familiar with it, so it's likely my SS code conflicts

**Comment by ekawahyu** [comment]
Here is the link to my Arduino-nRF5 fork with added SoftwareSerial library, and WInterrupts.c patch:

https://github.com/ekawahyu/arduino-nRF5/tree/test/softwareserial

I use port 0 (RX) and 1(TX) with constant data coming in from GPS. When I built SoftwareSerialExample sketch without SoftDevice, I can see all GPS data coming in just fine at 9600 bps. But once I added SoftDevice, then all GPS data become gibberish. What do you think causing it?

**Comment by ekawahyu** [comment]
I still have no idea how to fix this thing. I don't see anything wrong with the code, unless there is issue with interrupt priority?

**Comment by micooke** [comment]
The only thing I can see is that your code seems to be missing the changes to attachInterrupt that i proposed here. https://github.com/sandeepmistry/arduino-nRF5/pull/205

**Comment by ekawahyu** [comment]
You mean the changes in WInterrupt.c and its header? I have it here: https://github.com/ekawahyu/arduino-nRF5/tree/micooke-pr205

**Comment by micooke** [comment]
Ah, yep. Sorry then I'm not sure. Have you tried other pins incase the Softdevice is using those for something else? I'm not much use at the moment, sorry, but I don't have access to my Arduino devices at the moment

**Comment by ekawahyu** [comment]
yeah, I tried D0-D5 with no luck, I may have to use a breakout board and try it again, making sure that there is absolutely no peripheral attached to those pins. But the thing is, it works when I build the example without S132. But when the S132 is in, then all I can see is the gibberish output. Really weird.
